### PR TITLE
Fixed #429 & #430: Adding status icon & link to block height.

### LINKED ISSUE #429: Link block height to the corresponding block
image
For the places we show block height, make the block height clickable and link it to the corresponding block.

### PR DESCRIPTION
**Pull request by nmhung1210**
**What type of PR is this?**
bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #429 #430 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
This PR contains the following changes on UI:

**1. Status icons:**

image
image

**2. Link to block height**
image

### CODE REVIEW

File: src/shared/block/block-detail.tsx [hunk 1]
```diff
@@ -286,8 +286,21 @@ export function renderValue(text: string, record: any): JSX.Element | string {
...
-    case "status":
-      return <span>{parseInt(text, 10) === 1 ? "success" : "failure"}</span>;
+    case "height":
+    case "blkHeight":
+      return <FlexLink path={`/block/${text}`} text={text} />;
+    case "status": {
+      const success = parseInt(text, 10) === 1;
+      const iconName = success ? "check-circle" : "close-circle";
+      const color = success ? "#52c41a" : "#f5222d";
+      const statusText = success ? " Success" : " Failure";
```
Comment (CoderZhi) [inline src/shared/block/block-detail.tsx]: Put "Success" and "Failure" into translation.

File: src/shared/block/block-detail.tsx [hunk 2]
```diff
@@ -286,8 +286,21 @@ export function renderValue(text: string, record: any): JSX.Element | string {
...
       return <FlexLink path={`/block/${text}`} text={text} />;
-    case "status":
-      return <span>{parseInt(text, 10) === 1 ? "success" : "failure"}</span>;
+    case "height":
+    case "blkHeight":
+      return <FlexLink path={`/block/${text}`} text={text} />;
+    case "status": {
+      const success = parseInt(text, 10) === 1;
+      const iconName = success ? "check-circle" : "close-circle";
+      const color = success ? "#52c41a" : "#f5222d";
```
Comment (puncsky) [inline src/shared/block/block-detail.tsx]: use colors from iotex-explorer/src/shared/common/styles/style-color.tsx

Comment (nmhung1210) [reply src/shared/block/block-detail.tsx]: done

File: src/shared/block/block-detail.tsx (hunk 1 above)
Comment (nmhung1210) [reply src/shared/block/block-detail.tsx]: done

**Pull request merged**